Mathlib/Algebra/BigOperators/Order.lean

```diff
@@ -191,13 +190,11 @@ theorem prod_eq_one_iff_of_one_le' :
 #align finset.prod_eq_one_iff_of_one_le' Finset.prod_eq_one_iff_of_one_le'
 #align finset.sum_eq_zero_iff_of_nonneg Finset.sum_eq_zero_iff_of_nonneg
 
-@[to_additive existing sum_eq_zero_iff_of_nonneg]
+@[to_additive sum_eq_zero_iff_of_nonpos]
 theorem prod_eq_one_iff_of_le_one' :
```

Comment:
Yeah - the lemma directly below this one is named that, with a different spelling for the condition

Reply:
Oh I see: I didn't find the other lemma because I was looking in this branch, which does not contain the recent changes from #7372.

I note however that lemma is called `Fintype.prod_eq_one_iff_of_le_one` making room for this to be called `Finset.prod_eq_one_iff_of_le_one` which I think actually makes most sense.

Reply:
Good point!